README.rst

```diff
@@ -41,6 +41,29 @@ project's virtual environment.
 
 .. _virtual environment: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
 
+
+Cross-environment usage of ``requirements.in``/``requirements.txt`` and ``pip-compile``
+=======================================================================================
+
+The dependencies of a package can change depending on the Python environment in which it is installed.
```

Comment:
How about an example here, just to make it clear what we are talking about:

Consider this ```requirements.in```:

```
enum34; python_version < '3.4'
distro; platform_system == 'Linux'
```

Maybe not needed, but shouldn't hurt to show some real "code".

Reply:
Uh, scratch this. I was thinking about self-added environment markers now only, but this example would be just misleading. So nevermind.